src/helpers/buildWalletSections.js

```diff
@@ -408,7 +406,7 @@ const buildImagesToPreloadArray = (family, index, families) => {
       return {
         id: uniqueId,
         priority,
-        uri: svgToPngIfNeeded(getLowResUrl(image_url)),
```

Comment:
and what happens to this thing? Why did you remove it? Is it handled in another way already?

Reply:
meant to leave a comment, this happens further down in the actual image component so its unneeded here.

a lot of the changes I made we're to condense all of this logic to the actual image components